graylog2-web-interface/src/components/metrics/MetricsList.jsx

```diff
@@ -4,14 +4,14 @@ import styled from 'styled-components';
 
 import { Metric } from 'components/metrics';
 
-const MetricListWrap = styled.ul`
+const MetricListWrap = styled(({ theme }) => `
```

Comment:
Legit! 
Just a small thing. The metrics page is crashing because of missing element. 
It should be `styled.ul(({ theme }) => `  instead of `styled(({ theme }) => ``